src/components/OrdersWidget/OrderRow.tsx

```diff
@@ -264,18 +443,23 @@ const OrderRow: React.FC<Props> = props => {
   return (
     <>
       {sellToken && buyToken && (
-        <OrderRowWrapper className={'orderRow' + (pending ? ' pending' : '')}>
-          <PendingLink pending={pending} transactionHash={transactionHash} />
-          <DeleteOrder
-            isMarkedForDeletion={isMarkedForDeletion}
-            toggleMarkedForDeletion={toggleMarkedForDeletion}
-            pending={pending}
-            disabled={disabled}
-          />
+        <OrderRowWrapper className={'orderRow' + (pending ? ' pending' : '')} $open={openCard}>
+          {pending ? (
+            <PendingLink pending={pending} transactionHash={transactionHash} />
+          ) : (
+            <DeleteOrder
+              isMarkedForDeletion={isMarkedForDeletion}
+              toggleMarkedForDeletion={toggleMarkedForDeletion}
+              pending={pending}
```

Comment:
Since this component won't be rendered when `pending` you can also remove from its props

Reply:
:+1: